### ISSUE TITLE
Presentation Podium (S)

### ISSUE THREAD

**Issue by twaldorf** [FrankyNeedles/DAOHouse#5]
## Requirements
- The control panel allows users to pick a proposal to display on the screen via a 2D UI showing a list of all proposals

## Limitations
- Tris limit:
- Texture budget

**Comment by lilyannc** [comment]
***CURRENTLY UNTEXTURED***
Reference image used to model:
![image](https://user-images.githubusercontent.com/66533253/141393366-b20f85c4-7d69-4e9a-8ddc-604aae8404b7.png)

Podium angles with imported character for scale reference
![image](https://user-images.githubusercontent.com/66533253/141393044-d172c423-9db8-4754-8f0f-0cb26589381b.png)
![image](https://user-images.githubusercontent.com/66533253/141393068-b186b44e-8c70-402a-ad5a-ec751c392bca.png)
![image](https://user-images.githubusercontent.com/66533253/141393094-5984e3b3-7af6-44a4-8b5c-e354c2527ff6.png)

Added material on the bottom ring to make it cyan. Will be made to glow later. Is it possible to make it glow only when someone is standing on the podium?
![image](https://user-images.githubusercontent.com/66533253/141393292-ba7440d6-a31c-4fe3-a3bd-4abadbe10e8e.png)

Possible ideas for buttons for scrolling between proposals and open 2D UI:
![image](https://user-images.githubusercontent.com/66533253/141392875-ba6191d2-969b-4950-97df-a015db8f1a23.png)

**Comment by twaldorf** [comment]
@lilyannc Yes, I think it can glow when someone is standing on it—it'll be a material component add/remove, can be done through code as long as the object has both materials with it.

What do you need to do the textures here? Want me to send the textures used for the structures down below?

**Comment by lilyannc** [comment]
Perfect!

The textures from the structures below would be perfect for this to give it a stone structure look.

Are the buttons good or should they be changed in any way? I figured I would put a mini UI texture onto the square button that opens the UI once the 2D UI gets made.

**Comment by twaldorf** [comment]
@lilyannc the textures are actually all in this repo, here https://github.com/FrankyNeedles/DAOHouse/tree/master/models

Let's get rid of the buttons for now, I think those should be sprites because they will need to change with the code.

**Comment by lilyannc** [comment]
![image](https://user-images.githubusercontent.com/66533253/145656748-63c2b37b-84c8-4144-9d99-c4e1b1357771.png)

![image](https://user-images.githubusercontent.com/66533253/145656757-5f3b3e58-36d5-4449-9193-5bb6850d085f.png)

If I don't say where texture stretching as occurred on this podium, is it obvious where it really is?

**Comment by twaldorf** [comment]
moving this to review, as I am putting it in the scene now

Payout of $200 USD issued 15 December 2021 via USD
